README.md

```
@@ -319,16 +363,102 @@ With the default configuration (`base_dir: "../worktrees"`):
+    │   └── payment/        # wtp add feature/payment
+    └── hotfix/
+        └── bug-123/        # wtp add hotfix/bug-123
+```
+
+**Namespacing benefits:**
+- Prevents conflicts when multiple projects share `../worktrees`
+- Branch names with slashes are preserved as directory structure
+- Automatically organizes worktrees by project and type/category
+
+**Legacy layout** (without namespacing):
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)